.github/workflows/merge-conflicts.yml

```diff
@@ -1,18 +1,17 @@
-name: "Merge Conflicts"
+name: 'Merge Conflicts'
 
 on:
   push:
     branches:
       - master
-  pull_request:
+  pull_request_target:
     types:
       - synchronize
 jobs:
   triage:
     runs-on: ubuntu-latest
-    if: github.repository == 'jellyfin/jellyfin-web'
```

Comment:
Is this not needed to prevent running on forks?

Reply:
Yes it is. I 'd say it should be kept.

Reply:
@thornbill @nielsvanvelzen Not needed with ``pull_request_target`` trigger